src/pages-chibi/implementations/Anizium/main.ts

```diff
@@ -0,0 +1,151 @@
+import { PageInterface } from '../../pageInterface';
+
+export const Anizium: PageInterface = {
+  name: 'Anizium',
+  type: 'anime',
+  domain: 'https://anizium.co',
+  languages: ['Turkish'],
+  urls: {
+    match: ['*://anizium.co/*', '*://www.anizium.co/*', '*://m.anizium.co/*'],
+  },
+  search: 'https://anizium.co/search?q={searchtermPlus}',
+  sync: {
+    isSyncPage($c) {
+      return $c
+        .and(
+          $c.url().contains('/watch/').run(),
+          // Ensure we have valid episode data
+          $c
+            .or(
+              $c.url().urlParam('episode').boolean().run(),
+              $c.querySelector('.episode-info').boolean().run(),
+            )
+            .run(),
+        )
+        .run();
+    },
+    getTitle($c) {
+      const element = $c.querySelector('.trending-text');
+      return $c
+        .coalesce(
+          // Primary: trending-text selector with cleaning
+          element.text().trim().replaceRegex('\\s+S\\d+\\s+B\\d+.*', '').trim().run(),
+          // Fallback 1: page title cleaning
+          $c.title().replaceRegex(' - Anizium.*', '').trim().run(),
+          // Fallback 2: meta og:title
+          $c.querySelector('meta[property="og:title"]').getAttribute('content').run(),
+          // Fallback 3: h1 title
+          $c.querySelector('h1').text().trim().run(),
+        )
+        .ifNotReturn()
+        .run();
+    },
+    getIdentifier($c) {
+      // Universal season-aware identifier for all anime
+      const baseId = $c.url().urlPart(4);
+      const season = $c.url().urlParam('season');
+
+      return $c
+        .if(
+          // If season parameter exists, create enhanced identifier
+          $c.and(baseId.boolean().run(), season.boolean().run()).run(),
+          // Format: "{animeId}?s={season}" - separate identifier for each season
+          baseId.concat('?s=').concat(season.string().run()).run(),
+          // Fallback: base ID only
+          baseId.run(),
+        )
+        .run();
+    },
+    getOverviewUrl($c) {
+      // Get base ID (without season parameter)
+      const baseId = $c.url().urlPart(4);
+      return $c.string('https://anizium.co/anime/').concat(baseId.run()).run();
```

Comment:
Better use `$c.string('/anime/').concat(baseId.run()).urlAbsolute().run();` then we do not have an hardcoded domain